x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/index.tsx

```diff
@@ -357,6 +358,14 @@ export const AgentListPage: React.FunctionComponent<{}> = () => {
       render: (host: string, agent: Agent) => (
         <EuiLink href={getHref('fleet_agent_details', { agentId: agent.id })}>
           {safeMetadata(host)}
+          {agent.type === 'EPHEMERAL' && (
```

Comment:
@jen-huang @hbharding This will only display the badge for the `EPHEMERAL` type. Other options are 

  a) for all types (`PERMANENT`, `EPHEMERAL`, and `TEMPORARY`)
  b) Any type _except_  `PERMANENT` (`EPHEMERAL` and `TEMPORARY`)

Let me know if I should update this

Reply:
this logic looks good to me. btw, I didn't know we had `TEMPORARY` as an agent type. it's not used, so could we clean that up as part of this PR?